pkg/backend/display/events.go

```diff
@@ -18,6 +19,8 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 )
 
+var matchAnsiControlCodes = regexp.MustCompile(`\x1b\[[0-9;]*[mK]`)
```

Comment:
👍 
This looks like it'll cover at least all color cases.
We can expand this if we need to cover more escape codes in the future.